### DISCUSSION TITLE
Is it possible to specify the amount of products per vehicle?

### DISCUSSION THREAD

**Discussion by williamrijksen** [reinterpretcat/vrp#36] [Q&A]
Let's say we have a truck which can deliver 4 televisions. Is it possible to assign an amount of specific products to a vehicle and when it didn't fit it'll reload?

**Comment by reinterpretcat** [comment]
This sounds like regular demand/capacity property: specify `[4]` as capacity of a vehicle and `[1]` as demand of a job
